### ISSUE TITLE
Rename the Expungable Record Summary to just Record Summary

### ISSUE THREAD

**Issue by rebecca-draben** [deardurham/dear-petition#380]
Rename Expungable Record Summary to just Record Summary because not all of the records are expungable.  Definitely rename on UI.  Renaming in the code would be desirable as well.

**Comment by austin-bagwell** [comment]
@rebecca-draben I updated the front end components to read "Record Summary", but wasn't able to change the names in the back end without causing issues. 

I started with a find-and-replace for "expungable_summary", then went through the files it was found in to see if everything still lined up correctly.  Those were mostly in expungable_summary.py, viewsets.py, and test_expungable_summary.py. I must have missed something because I could not download the summary from the front end after the rename. 

Here's a screenshot of the error I was able to see after chaning those names. Do you have any suggestions? 
<img width="836" alt="generate_summay_rename_screenshot" src="https://github.com/deardurham/dear-petition/assets/113562625/96c17742-d3e9-428b-b70f-21b21bc68a50">

**Comment by rebecca-draben** [comment]
@austin-bagwell It looks like the front end is using a GET request instead of a POST request when calling the back end web service.  If you change `def generate_expungable_summary(self, request, pk):` in `viewsets.py` to `def generate_summary(self, request, pk):` then you should change `url: `/batch/${batchId}/generate_expungable_summary/` to `url: `/batch/${batchId}/generate_summary/` in both `GenerationPage.jsx` and `ExistingPetitions.jsx`.  Both of those are already set up as POST requests, so I'm not sure how you are ending up with a GET request.  If you'd like, I can take a closer look at what you've got right now and we can figure it out.

**Comment by austin-bagwell** [comment]
@rebecca-draben  I think I got it 99% solved. The only thing I can't seem to change without breaking something is this line in records_summay.py:  `TEMPLATE = "expungable_summary.docx"`

As best I can tell, that is the name of template we have set up in DocxTemplate. I haven't figured out how to see those yet.

I ran the pytest stuff and everything passed. I'll go ahead and open the PR but if there's a way to adjust that TEMPLATE maybe we should change that too before merging,

<img width="752" alt="Screenshot 2023-05-12 at 8 32 27 PM" src="https://github.com/deardurham/dear-petition/assets/113562625/f11ec520-42b2-435c-af47-c73b30037681">
[Records Summary (11).docx](https://github.com/deardurham/dear-petition/files/11468512/Records.Summary.11.docx)
